### PR TITLE
Updated Mado Gear effects

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: #2099

* **Server Mode**: Pre-Renewal and Renewal

* **Description of Pull Request**: 
Updated Mado Gear bonuses from kRO patch: [here](http://ro.gnjoy.com/news/devnote/View.asp?category=1&seq=3652742&curpage=1)

TODO:
- [x] Add Emergency Magic Gear.
  - [x] Add Emergency Magic Gear to NPC.
- [x] Natural HP recovery increased.
- [x] Walkspeed at level 5 increased.
- [x] Increased maximum weight by 1500.
- [x] Allowed use of Teleport skill.
- [x] Chemical Protection added benefits.
- [x] Update Overheat limit and interaction with gear.

### CODE REVIEW

File: src/map/status.c [hunk 1]
```diff
@@ -3176,6 +3176,8 @@ bool status_calc_weight(struct map_session_data *sd, enum e_status_calc_weight_o
 			sd->max_weight += sd->max_weight * sc->data[SC_KNOWLEDGE]->val1 / 10;
 		if ((skill = pc_checkskill(sd, ALL_INCCARRY)) > 0)
 			sd->max_weight += 2000 * skill;
+		if (pc_ismadogear(sd))
+			sd->max_weight += 1500;
```
Comment (Valorro) [inline src/map/status.c]: @aleos89 i tried to apply this, seems only increase 150 weight ingame
maybe this? `sd->max_weight += 15000;`

Comment (aleos89) [inline src/map/status.c]: Thanks! Corrected that.

**Comment by ecdarreola** [comment]
How to fix conflicting files?

**Comment by ecdarreola** [comment]
Does this resolve some of the skills item requirement not needed to be equipped for them to use it?

**Comment by Jeybla** [comment]
@ecdarreola sounds like you're looking for #2374. It's already been merged to the master.

**Comment by ecdarreola** [comment]
@Jeybla I'm confused now. In iRO some of Mado skills doesn't need the item to be equipped while on kRO it needs to be equipped?

**Comment by aleos89** [comment]
@ecdarreola You can't really compare iRO and kRO, especially on 3rd class skills. iRO customized a lot of them for one, and two, not all Mechanic skills require the equipment to be on while some skills do.

**Comment by Badarosk0** [comment]
@aleos89 look this:
https://www.youtube.com/watch?v=9yeb2j1VFlQ

Shows using skill without needing to equip item.

**Comment by aleos89** [comment]
Which skill and what minute?

**Comment by Badarosk0** [comment]
https://youtu.be/9yeb2j1VFlQ?t=111

**Comment by aleos89** [comment]
This PR has those fixes.

**Pull request merged**